core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/LegacyYouTubePlayerView.kt

```diff
@@ -88,11 +88,13 @@ internal class LegacyYouTubePlayerView(
    * @param handleNetworkEvents if set to true a broadcast receiver will be registered and network events will be handled automatically.
    * If set to false, you should handle network events with your own broadcast receiver.
    * @param playerOptions customizable options for the embedded video player, can be null.
+   * @param videoId used to load an initial video, can be null.
```

Comment:
Rephrase to: @param optional videoId, used to load a video right after initialization.

Reply:
Do you mean "`@param` videoId optional, used to load a video right after initialization."? — the variable name must come directly after `@param` in javadoc syntax.

Reply:
Yes, you're right :)